notifications.json

```diff
@@ -0,0 +1,478 @@
+                    },
+                    "status": {
+                        "type": "integer",
+                        "default": 403
+                    }
+                }
+            }
+        },
+        "trait:apiErrors:404": {
+            "description": "",
+            "schema": {
+                "type": "object",
+                "description": "Object Not Found.",
+                "properties": {
+                    "error": {
+                        "type": "string"
+                    },
+                    "message": {
+                        "type": "string"
+                    },
+                    "status": {
+                        "type": "integer"
+                    }
+                }
+            }
+        },
+        "trait:apiErrors:429": {
+            "description": "",
+            "schema": {
+                "type": "object",
+                "description": "Quota excedeed.",
+                "properties": {
+                    "error": {
+                        "type": "string",
+                        "default": "API Error"
+                    },
+                    "message": {
+                        "type": "string"
+                    },
+                    "status": {
+                        "default": 429,
+                        "type": "integer"
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Content": {
+            "title": "Content",
+            "type": "object",
+            "properties": {
+                "title": {
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 255
+                },
+                "body_text": {
+                    "type": "string",
+                    "faker": "lorem.paragraph"
+                },
+                "body_html": {
+                    "type": "string",
+                    "faker": "lorem.paragraph"
+                }
+            },
+            "required": [
+                "title"
+            ]
+        },
+        "MessageDefinition": {
+            "title": "MessageDefinition",
+            "type": "object",
+            "properties": {
+                "to": {
+                    "description": "One fiscal code (CF).\n",
+                    "type": "string"
+                },
+                "dry_run": {
+                    "type": "boolean",
+                    "description": "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+                },
+                "time_to_live": {
+                    "type": "integer",
+                    "description": "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 4 weeks, and the default value is 4 weeks.",
+                    "minimum": 0
```

Comment:
these values are taken from the firebase cloud messaging API, obviously we can change them.

maybe 1 year it's too much.